feature-utils/poly-look/src/react-components/filterChips/filterChips.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useState } from "react";
+import * as propTypes from "prop-types";
+console.log("propes", propTypes);
+import Chip from "../chip/chip.jsx";
```

Comment:
Do you think it's worth the while to have different directories? After all, they're functionally dependent.

Reply:
Do you mean the import in line 2? That was some provisional solution for the building error, but I'll check that again.

Reply:
No, line 4; as long as you're refactoring, having "chip" separated from "filterchips" is probably not worth the while.

Reply:
My idea was creating a different component for Chip to be able to use it individually/independently. Are you suggesting to move the Chip component back to FilterChips?

Reply:
That, or just put the two files in a single directory, without their individual directories.